src/application/components/Alert.tsx

```diff
@@ -0,0 +1,46 @@
+import type { VariantProps } from "class-variance-authority";
+import { cva } from "class-variance-authority";
+import type { ElementType, ReactNode } from "react";
+import type { OmitNull } from "../types/utils";
+import IconErrorSolid from "@apollo/icons/large/IconErrorSolid.svg";
+import { twMerge } from "tailwind-merge";
+
+type Variants = OmitNull<Required<VariantProps<typeof alert>>>;
+
+interface AlertProps extends Variants {
+  children?: ReactNode;
+  className?: string;
+}
+
+const alert = cva(
+  ["px-4", "py-3", "border-l-4", "rounded", "flex", "gap-2", "items-start"],
+  {
+    variants: {
+      variant: {
+        error: [
```

Comment:
For now we only need the `error` variant, so I've only implemented this one. We can add others as we need them.